CHANGELOG.md

```diff
@@ -2,6 +2,55 @@
 
 [comment]: <> (## Unreleased Changes)
 
+### FEATURES
+
+- <high level summary>
+
+- [modules]
```

Comment:
We should use the actual crate names instead of the folder names, ie. ibc, ibc-relayer, ibc-relayer-cli